README.md

```diff
@@ -11,13 +10,15 @@ Options for getting started:
 * Clone the repo: `git clone https://github.com/BrightspaceHypermediaComponents/siren-sdk.git`.
 * Install with [npm](https://www.npmjs.com/): `npm install siren-sdk`
 
-## Versioning, Releasing & Deploying
+## Versioning & Releasing
```

Comment:
This is a switch to `semantic-release` for releasing, which I'm assuming Maya/Vincent you're OK with? Let me know if not and I can switch it back to the incremental release method.